### ISSUE TITLE
Do you have plans to update with content from 12/31/20 driver pack?

### ISSUE THREAD

**Issue by scyto** [daveschafer/NUC10i7FN-WindowsServer#1]
Thanks for putting this together, do you plan to update for 12/18/20 Intel-NUC-Kit-NUC7i7BNH_Win10-64_Drivers.20201231 

I found that for the ethernet one can just manually 'have disc' to the pro set software.
![image](https://user-images.githubusercontent.com/11369180/103446931-d9736f80-4c39-11eb-9b80-f9765cb5e161.png)
  it will warn you it isn't tested but works ok, no edits needed.

To my surprise doing a search into the folder structure in that new intel driver pack (Intel-NUC-Kit-NUC7i7BNH_Win10-64_Drivers.2020123) loaded the thunderbolt controller driver - do i need to do more. 

still tarting around with the other drivers, will contribute to repo when i have everything installed.

tl;dr i have ethernet and thunderbolt controller loaded without disabling driver signing or modifying drivers.

**Comment by scyto** [comment]
link (no idea why pasting from one note is a PNG and not rich text) https://downloadcenter.intel.com/download/22283/Intel-Ethernet-Adapter-Complete-Driver-Pack

**Comment by scyto** [comment]
for wifi. no edit needed just:
![image](https://user-images.githubusercontent.com/11369180/103447073-5226fb80-4c3b-11eb-83c8-bcebe5df96fc.png)

hmm may have spoken to fast, while the device is working in device manager and in network control panel, it isn't finding nay networks...

**Comment by scyto** [comment]
oh i am effing idiot I have a NUC10i7FNH1 like you but downloaded a different pack, ignore me for now (though the thunderbolt driver from the NUC7 pack did load, just need to test if it works i guess.)

**Comment by scyto** [comment]
ok summary, using https://downloadcenter.intel.com/product/188811/Intel-NUC-10-Performance-kit-NUC10i7FNH
I managed to get ALL drivers installed except BT PAN with no need to edit any inf files and no need to disable driver signing.
I did have to do the manual pathing trick for the NIC, for rest i searched the set of drivers uncompressed.
Graphics installed ok, thunderbolt appears to have installed ok (will test devices later).

**Comment by daveschafer** [comment]
These are some really helpfull insights, I will check the update procedure on my NUC and will update this repo if everything succeeds

**Comment by scyto** [comment]
glad i could contribute even if this was a stream of consiousness, i also installed the audio drivers and enabled the audio service

fwiw these are my rough notes
![image](https://user-images.githubusercontent.com/11369180/103447544-7be32100-4c41-11eb-8aa4-b1deba26d963.png)

**Comment by scyto** [comment]
BTW thanks for your reddit article, it is why i took a risk buying two of these a week ago for two hyper-v nodes.
I am not sure i want to try these driver shenanigans on hyper-v server so probably will stick with data center and desktop mode.

**Comment by scyto** [comment]
I created a complete set of instructions, optimized the workflow (no need to download the generic NIC driver pack, it is the bundle).   Also added some notes for server core.  But as i found hyper-V server and data center only had a memory footprint difference of about 700MB i decided to stick with the GUI version of data center server https://www.reddit.com/r/intelnuc/comments/kp7x5t/windows_server_nuc_nuc10i7fnx_getting_all_drivers/

I even solved the BT PAN because of my OCD (but, like who the fuck uses BT PAN for anything? lol!)

**Comment by daveschafer** [comment]
Well written instructions, I will update the repo in time. My vision would be a simple installer script without the need to pull custom drivers from an external repository (like mine). I remember back in the days it was much easier to getting Windows Server on a NUC (NUCi6), just had to manually copy the NIC driver from a thumb drive and the rest could be installed with the (now discontinued) intel driver management toolkit.

Also I just noticed that some drivers (but definitely not all) are getting driver updates through windows updates.

Just a little side note, I actually run hyper-v on 2 NUCS in a cluster (Nuc10i and Nuc6i) and so far I never experienced any problems (running almost a year now). I even got RemoteFX working for some graphical stuff (but that is another story).
